### PR TITLE
Add Omron FINS protocol dissector

### PR DESCRIPTION
**Pull request by 0xA50C1A1**
Please sign (check) the below before submitting the Pull Request:

- [x] I have signed the ntop Contributor License Agreement at https://github.com/ntop/legal/blob/main/individual-contributor-licence-agreement.md
- [x] I have read the contributing guide lines at https://github.com/ntop/nDPI/blob/dev/CONTRIBUTING.md
- [x] I have updated the documentation (in doc/) to reflect the changes made (if applicable)

Describe changes:

Factory Interface Network Service (FINS) is a network protocol used by Omron PLCs.

### CODE REVIEW

File: tests/cfgs/default/result/fins.pcap.out [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	2	TCP 10.1.1.173:17134 <-> 10.1.1.164:9600 [proto: 362/FINS][IP: 0/Unknown][ClearText][Confidence: DPI][DPI packets: 4][cat: IoT-Scada/31][6 pkts/385 bytes <-> 4 pkts/366 bytes][Goodput ratio: 13/40][0.59 sec][bytes ratio: 0.025 (Mixed)][IAT c2s/s2c min/avg/max/stddev: 2/145 117/151 158/157 58/6][Pkt Len c2s/s2c min/avg/max/stddev: 54/54 64/92 83/176 11/50][PLAIN TEXT (EL20DR)][Plen Bins: 75,0,0,25,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]
+	3	TCP 10.1.1.173:61717 <-> 10.1.1.164:9600 [proto: 362/FINS][IP: 0/Unknown][ClearText][Confidence: Match by port][DPI packets: 4][cat: IoT-Scada/31][1 pkts/58 bytes <-> 3 pkts/174 bytes][Goodput ratio: 0/0][8.83 sec][Plen Bins: 0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]
```
Comment (IvanNardi) [inline tests/cfgs/default/result/fins.pcap.out]: one

File: tests/cfgs/default/result/fins.pcap.out [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+	4	UDP 10.1.1.173:54855 <-> 10.1.1.164:9600 [proto: 362/FINS][IP: 0/Unknown][ClearText][Confidence: DPI][DPI packets: 1][cat: IoT-Scada/31][1 pkts/55 bytes <-> 1 pkts/148 bytes][Goodput ratio: 23/71][0.16 sec][PLAIN TEXT (EL20DR)][Plen Bins: 50,0,0,50,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]
+	5	UDP 10.1.1.173:59516 -> 10.1.1.164:9600 [proto: 362/FINS][IP: 0/Unknown][ClearText][Confidence: Match by port][DPI packets: 1][cat: IoT-Scada/31][1 pkts/42 bytes -> 0 pkts/0 bytes][Goodput ratio: 0/0][< 1 sec][Risk: ** Unidirectional Traffic **][Risk Score: 10][Risk Info: No server to client traffic][Plen Bins: 0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]
```
Comment (IvanNardi) [inline tests/cfgs/default/result/fins.pcap.out]: two

File: tests/cfgs/default/result/fins.pcap.out [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+	6	UDP 10.1.1.173:59517 -> 10.1.1.164:9600 [proto: 362/FINS][IP: 0/Unknown][ClearText][Confidence: Match by port][DPI packets: 1][cat: IoT-Scada/31][1 pkts/42 bytes -> 0 pkts/0 bytes][Goodput ratio: 0/0][< 1 sec][Risk: ** Unidirectional Traffic **][Risk Score: 10][Risk Info: No server to client traffic][Plen Bins: 0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]
```
Comment (IvanNardi) [inline tests/cfgs/default/result/fins.pcap.out]: three: could you remove these 3 flows? They are basically "empty"...

**Review by IvanNardi — CHANGES_REQUESTED** [review]
Just a minor issue

**Comment by 0xA50C1A1** [comment]
> Just a minor issue

Ok, I've deleted those empty flows

**Review by IvanNardi — APPROVED** [review]

**Comment by IvanNardi** [comment]
Thank you very much!

**Pull request merged**